frontends/web/src/routes/account/send/utxos.tsx

```diff
@@ -51,12 +53,29 @@ export class UTXOsClass extends Component<Props, State> {
     selectedUTXOs: {},
   };
 
-  public componentDidMount() {
+  private subscribtions: UnsubscribeList = [];
+
+  private subscribe() {
+    this.subscribtions.push(
+      syncdone(this.props.accountCode, () => this.getUTXOs()),
+    );
+  }
+
+  private getUTXOs() {
     accountApi.getUTXOs(this.props.accountCode).then(utxos => {
       this.setState({ utxos });
     });
   }
 
+  public componentDidMount() {
+    this.getUTXOs();
+    this.subscribe();
+  }
+
+  public componentWillUnmount() {
+    unsubscribe(this.subscribtions);
+  }
+
```

Comment:
Tiny nit: I find it more readable if `subscribe` and `getUTXOs` was defined after the React lifecycle methods   `componentDidMount`, `componentWillUnmount`